app/javascript/react/components/ApplicantTracker.jsx

```diff
@@ -124,101 +134,106 @@ export default function ApplicantTracker({
-                ? getFrenchFormatDateString(lastEmailInvitationSentAt)
-                : "-"}
-            </p>
-          )}
-          {showPostalInvitation && (
-            <p className={cssClassForInvitationDate("postal")}>
-              {lastPostalInvitationSentAt
-                ? getFrenchFormatDateString(lastPostalInvitationSentAt)
-                : "-"}
-            </p>
-          )}
-        </div>
-        <div className="row d-flex justify-content-around align-items-center">
-          <div className="col-3" />
-          {showSmsInvitation && (
-            <div className="col-3">
-              <button
-                type="button"
-                disabled={
-                  isLoading.smsInvitation ||
-                  rdvs.length > 0 ||
-                  !applicant.phone_number ||
-                  applicantStatus === "resolved"
-                }
-                className="btn btn-blue"
-                onClick={() => handleClick("smsInvitation")}
-              >
-                {isLoading.smsInvitation && "Invitation..."}
-                {!isLoading.smsInvitation && lastSmsInvitationSentAt && "Relancer"}
-                {!isLoading.smsInvitation && !lastSmsInvitationSentAt && "Inviter"}
-              </button>
-            </div>
-          )}
-          {showEmailInvitation && (
-            <div className="col-3">
-              <button
-                type="button"
-                disabled={
-                  isLoading.emailInvitation ||
-                  rdvs.length > 0 ||
-                  !applicant.email ||
-                  applicantStatus === "resolved"
-                }
-                className="btn btn-blue"
-                onClick={() => handleClick("emailInvitation")}
-              >
-                {isLoading.emailInvitation && "Invitation..."}
-                {!isLoading.emailInvitation && lastEmailInvitationSentAt && "Relancer"}
-                {!isLoading.emailInvitation && !lastEmailInvitationSentAt && "Inviter"}
-              </button>
-            </div>
-          )}
-          {showPostalInvitation && (
-            <div className="col-3">
-              <button
-                type="button"
-                disabled={
-                  isLoading.postalInvitation ||
-                  rdvs.length > 0 ||
-                  !applicant.address ||
-                  applicantStatus === "resolved"
-                }
-                className="btn btn-blue"
-                onClick={() => handleClick("postalInvitation")}
-              >
-                {isLoading.postalInvitation && "Invitation..."}
-                {!isLoading.postalInvitation && lastPostalInvitationSentAt && "Recréer"}
-                {!isLoading.postalInvitation && !lastPostalInvitationSentAt && "Inviter"}
-              </button>
-            </div>
-          )}
+    <div className="d-flex justify-content-around text-center flex-wrap mb-4 pb-3 tracking-blocks-wrapper">
+      {!showNotification && (
```

Comment:
Est-ce qu'à la place de `!showNotification`, on pourrait pas déclarer une variable `showInvitations = showSmsInvitation || showEmailInvitation || showPostalInvitation` en haut du component?
Le fait que l'on montre pas les invits quand on notifie c'est un side effect, mais on devrait peut-être pas baser la logique d'affichage dessus.
WDYT?

Reply:
yes why not